debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/pgoutput/PgOutputMessageDecoder.java

```diff
@@ -393,7 +394,7 @@ private void decodeUpdate(ByteBuffer buffer, TypeRegistry typeRegistry, Replicat
 
         Optional<Table> resolvedTable = resolveRelation(relationId);
         if (!resolvedTable.isPresent()) {
-            processor.process(null);
+            processor.process(new NoopMessage(transactionId, commitTimestamp));
```

Comment:
In which case would no table be found? Could you add a comment?

Reply:
@gunnarmorling Javadoc added